guides/doc-Monitoring_Project/topics/proc_installing-pcp-packages.adoc

```diff
@@ -0,0 +1,35 @@
+[id='installing-pcp-packages_{context}']
+= Installing PCP Packages
+
+This procedure describes how to install the PCP packages.
+
+.Prerequisites
+
+* Ensure you have a minimum of 20 GB space available in the `/var/log/pcp` directory.
++
+The default PCP data retention policy is to retain only that data collected during the past 14 days. Data storage per day is estimated to use usually between 100 MB and 500 MB of disk space, but may use up to several gigabytes.
```

Comment:
Feels like https://github.com/theforeman/foreman-documentation/pull/2407/files#diff-5673b605286a768dc0072be5f340b04306309d7ebc3da5e4d4369ef1a96792c4 should be included here or linked.